chirp/wxui/config.py

```diff
@@ -158,6 +158,16 @@ def get_bool(self, key, section=None, default=False):
     def set_bool(self, key, value, section=None):
         self.set(key, str(bool(value)), section)
 
+    def get_str_list(self, key, section=None, default=[]):
+        val = self.get(key, section)
+        if val is None:
+            return default
+        else:
+            return val.split(',')
+
+    def set_str_list(self, key, value, section=None):
+        self.set(key, ','.join(value), section)
```

Comment:
You need to make sure the value has no commas in it here, else it will not store/load the same value.